stylist/models.py

```diff
@@ -96,6 +96,30 @@ def compile_attrs(self):
             os.remove(custom_vars.name)
 
 
+class Font(models.Model):
+    """
+    Model for Fonts
+    """
+
+    family = models.CharField(_("Font Family"), max_length=50)
+    href = models.URLField(_("URL"), max_length=200)
```

Comment:
My only question is should the field be named "uri", "url" or "href"?  Not a big deal but wanted to pose that question.

Reply:
so here I'm just matching attribute name in the `<style>` tag that will be using it. @renderbox